content/4_Gold/TopoSort.mdx

```diff
@@ -927,6 +927,63 @@ public class Main {
 ```
 
 </JavaSection>
+<PySection>
+
+```py
+from collections import defaultdict, deque
+
+
+def main():
```

Comment:
why `main()`

Reply:
This is an optimization to speed up python code. The local variable lookup will be much faster compare to global variable lookup. Please refers to this [link](https://stackoverflow.com/questions/12590058/performance-with-global-variables-vs-local) to get more details.

Reply:
then add a doc comment explaining that

Reply:
Done!